string vehicle_name
int32 team_id
int32 phase_id
float64[] start
float64[] end        #stride of 2
#float64[] rewards   #stride of 3, (pos_x, pos_y, reward_value)
fast_task_planner/Reward[] rewards